src/view/translations/en.json

```
@@ -49,5 +49,54 @@
+    "toolbar-clue-temperature-sensor.title": "Temperature",
+    "toolbar-clue-temperature-sensor.description": "CLUE uses the BMP280 sensor, an environmental sensor with temperature.This precision sensor from Bosch is the best low-cost, precision sensing solution for measuring temperature with ±1.0°C accuracy.",
+    "toolbar-clue-temperature-sensor.tryItDescription": "You can set the temperature range from your code!",
+    "toolbar-clue-light-sensor.title": "Light/Color Sensor",
+    "toolbar-clue-light-sensor.description": "The light sensor uses APDS9960 and is able to return the red, blue, green and clear light.",
+    "toolbar-clue-light-sensor.tryItDescription": "You can set the color values of the light detected.",
+    "toolbar-clue-accelerometer-sensor.title": "Accelerometer",
+    "toolbar-clue-accelerometer-sensor.description": " The 3-axis accelerometer, can tell you which direction is down towards the Earth (by measuring gravity) or how fast the board is accelerating in 3D space.",
+    "toolbar-clue-accelerometer-sensor.tryItDescription": "Set the acceleration with the sliders.",
+    "toolbar-clue-led.title": "Neopixel",
+    "toolbar-clue-led.description": "The CLUE has one full RGB LED on it's back",
+    "toolbar-clue-led.tryItDescription": "Run your code and see the cool effects on the simulator!",
+    "toolbar-clue-a-b-push.title": "Buttons",
+    "toolbar-clue-a-b-push.description": "There are two buttons on the front of the CLUE (labelled A and B). The third button is to trigger both A and B buttons. You can detect when these buttons are pressed, allowing you to trigger code on the device.",
+    "toolbar-clue-a-b-push.tryItDescription": "Click them with your mouse or by pressing “A” and/or “B” on your keyboard!",
+    "toolbar-clue-gpio.title": "Input/Output (GPIO)",
+    "toolbar-clue-gpio.description": "On the bottom edge of your CLUE, you have 25 pins. These pins allow you to really get creative. You can create circuits, connect external things like buzzers and motors and make your own fun projects.",
+    "toolbar-clue-gpio.tryItDescription": "If you would like to see this sensor supported, please +1 the feature addition issue on GitHub!",
+    "toolbar-clue-sound-sensor.title": "Sound",
+    "toolbar-clue-sound-sensor.description": "The CLUE uses a PDM MEMS microphone. PDM is a little like 1-bit PWM. You clock the mic with a 1 MHz - 3 MHz clock rate, and on the data line you'll get a square wave out that syncs with the clock.",
+    "toolbar-clue-sound-sensor.tryItDescription": "If you would like to see this sensor supported, please +1 the feature addition issue on GitHub!",
+    "toolbar-clue-pressure-sensor.title": "Pressure",
+    "toolbar-clue-pressure-sensor.description": "CLUE uses the BMP280 sensor for barometric pressure. Because pressure changes with altitude, and the pressure measurements are so good, you can also use it as an altimeter with  ±1 meter accuracy.",

```

```suggestion
    "toolbar-clue-pressure-sensor.description": "CLUE uses the BMP280 sensor for barometric pressure. Because pressure changes with altitude, you can also use it as an altimeter with ±1 meter accuracy.",

```
I don't think having it say "so good" is smart without any evidence. I prefer it like ^

---

Okok, it was copied from the adafruit website ahaha